packages/components/package.json

```diff
@@ -107,6 +107,7 @@
     "classnames": "^2.2.6",
     "formik": "^1.5.8",
     "gh-pages": "^2.1.1",
+    "mobx-react": "^6.2.2",
```

Comment:
Is there a way to get around `MobxPropTypes` without adding `mobx-react` to components?

Reply:
I didn't find any. Btw there is a wish-list [here](https://github.com/mobxjs/mobx-react/issues/640) asking to separate type definitions from core package.
For now we can replace `MobxPropTypes` with simple `PropTypes` or just proceed with `mobx-react`.
What do you think?

Reply:
Yeah good idea normal PropTypes is fine for this imo (the components library could be used in other projects without mobx for example)